src/v/utils/tests/utf8_control_chars.cc

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2023 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#include "utils/utf8.h"
+
+#include <boost/test/unit_test.hpp>
+
+BOOST_AUTO_TEST_CASE(control_chars_default_thrower) {
+    {
+        const std::string_view good_string
+          = " !\"#$%&'()*+,-./"
+            "0123456789:;<=>?@ABCDEFHIJKLMNOPQRSTUVWXYZ[\\]^_`"
+            "abcdefghijklmnopqrstuvwxyz{|}~";
+        BOOST_CHECK_NO_THROW(validate_no_control(good_string));
+    }
+    for (char ch = 1; ch <= 0x1f; ch++) {
+        std::string_view const temp{&ch, 1};
```

Comment:
nit: be consistent with`std::string_view const` or `const std::string_view`
I think we typically do the latter but we don't statndardize